src/structures/session.ts

```diff
@@ -737,6 +774,10 @@ export default abstract class Session {
             return false;
         }
 
+        if (this.isGameSession() && this.isClipMode()) {
```

Comment:
Doesn't clip mode imply game session?

Reply:
also cant we just set this on line 705

Reply:
This also needs to be set when the round is starting, so it would need to be added to both of these branches:

```
        if (!round.songStartedAt) {
            switch (seekType) {
                case SeekType.BEGINNING:
                    seekLocation = 0;
                    break;
                case SeekType.MIDDLE:
                    seekLocation = songDuration * (0.4 + 0.2 * Math.random());
                    break;
                case SeekType.RANDOM:
                default:
                    seekLocation = songDuration * (0.6 * Math.random());
                    break;
            }
        } else if (clipAction === ClipAction.NEW_CLIP) {
            // Clip mode and the user requested another segment
            seekLocation = songDuration * (0.6 * Math.random());
```

I can move its initialization just below these if that makes it simpler